db/system_keyspace.hh

```diff
@@ -598,6 +616,10 @@ future<> set_bootstrap_state(bootstrap_state state);
 
     api::timestamp_type schema_creation_timestamp();
 
+/**
+ * Builds a mutation for system.clients containing provided connections.
+ */
+mutation make_client_mutation(client_data&& cd);
```

Comment:
This should probably be a local (static) function in the clientlist_virtual_reader module. No need to pollute `system_keyspace`'s header.